README.md

```diff
@@ -123,8 +123,159 @@ If you're wondering what a specific protocol does, your friend Google is always
 
 This is really all you need to know to get started in Wireshark, so lets start looking at some of the cool things you can do!
 
-## Basic Packet Analysis in Wireshark
+## Basic Packet Analysis, Sorting, and Filtering in Wireshark
+
+### Packet Analysis
+
+#### The Tip of The Iceberg
+
+Packets (also referred to as frames) contain a large amount of information, so it's always best to start with what we can see easily. Let's take a look at the _Packet Listing_ window and gather the information from there first. We are going to use `Basics.pcap` in this section.
+
+![](DNS_listing.png)  
+_Screenshot of the Packet Listing window_  
+
+We will be analyzing packet #3.
+Just by looking at all the sorting columns that correspond to our packet (row), we can gather the following information:
+
+- **Number**: 3  
+_The number of the packet in the capture file_
+- **Time**: 0.000041  
+_Absolute time since the beginning of the capture (in seconds)_
+- **Source**: 10.0.0.132  
+_The system that sent the packet. In most cases, the IPv4 address_
+- **Destination**: 10.0.0.2  
+_The system that received the packet. In most cases, the IPv4 address_
+- **Protocol**: DNS  
+_An application layer protocol that the packet used_
+- **Length**: 86  
+_Number of bytes in this packet_
+- **Info**: Standard query 0x332e A www.offensive-security.com  
+_Summary of the information in the highest layer protocol_
 
+This is the easiest way to see the most common information about the packets. However, there is much more that can be uncovered.
+
+#### Digging Deeper
+
+Glancing at the _Packet Listing_ is always nice, but it's rarely enough. If we want to learn more about the packet, we should click on it and explore the _Packet Details_. We have already discussed what information goes in each one of the layers (drop down rows) in the previous section so it will be easier to analyze our specific DNS packet, knowing that information. We will start from the top of the list with the physical layer, and work our way to the bottom - the "highest" protocol.
```

Comment:
"previous section, so it will be" (add comma)

Reply:
I think you can get rid of the comma before "knowing that information."

Reply:
And no need for a comma after "physical layer, and work"

Reply:
"knowing that information" sounds a little repetitive due to you already saying "We have already discussed..."